source/Objects/Towers/Upgrades/Upgrade.gd

```diff
@@ -0,0 +1,23 @@
+class_name Upgrade
+extends Node
+
+signal applied(cost)
+
+
+export var text := "Upgrade name"
+export(String, MULTILINE) var description := "Upgrade description"
+export var cost := 100
+export(float, EASE) var cost_curve := 1.0
```

Comment:
You have to use the `ease` function to use such a value. You're using it as a multiplier, so I went ahead and removed the hint + renamed the property.